### PR TITLE
eh-eventsProvider

### PR DESCRIPTION
**Pull request by egeehartz**
-check that events provider is filled out correctly (edit function commented out for future stretch goal usage)
-make an empty "events" array in your database
-fill out an event form to test the form
-delete the event to test the delete functionality
-check to make sure contentTargets and contentElements are targeting the correct containers (formContainer and eventsContainer)
-check to see if css files were added and imported to the main.css

(eventList() and eventForm() do need to called on main.js for tests)

### CODE REVIEW

**Review by Johnson4996 — APPROVED** [review]

File: src/scripts/Events/EventsList.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { getEvents, useEvents } from "./EventDataProvider.js"
+import {eventsHTML} from "./Events.js"
+
+
+
+
+const contentTarget = document.querySelector(".contentContainer")
```
Comment (skratz17) [inline src/scripts/Events/EventsList.js]: The EventsList contentTarget should be ".eventsContainer" within the contentContainer I think?

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { LoginForm } from "./auth/LoginForm.js"
-import { RegisterForm } from "./auth/RegisterForm.js"
-import { Nutshell } from "./Nutshell.js"
+//import { RegisterForm } from "./auth/RegisterForm.js"
```
Comment (skratz17) [inline src/scripts/main.js]: can you uncomment the Register and Nutshell imports, and move both the eventList and eventForm imports to `Nutshell.js` and then just call them there? That way the event stuff should appear right when a user logs in

File: src/scripts/Events/EventForm.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+
+
+        const newEvent = {
+            name: eventTitle,
+            date: eventDate,
+            city: eventCity,
+            state: eventState,
+            zip: eventZipCode,
+            //userId: //????
```
Comment (skratz17) [inline src/scripts/Events/EventForm.js]: This can be `userId: sessionStorage.getItem("activeUser")`

File: src/scripts/Events/EventsList.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+//const eventHub = document.querySelector(".container")
+
+
+const render = () => {
+    const events = useEvents()
+
+    const rep = events.map(event => {
+        return eventsHTML(event)
+    })
+    contentTarget.innerHTML = rep
```
Comment (skratz17) [inline src/scripts/Events/EventsList.js:16]: You never did a .join("") on the map so when this is rendered to innerHTML you get the commas showing up in the DOM

**Comment by skratz17** [comment]
OK oneeee more thing, because now you removed the `<form>` tag from `EventForm.js` (which was causing the page to refresh when you submitted the form), the EventList is now not updating when event state changes... so if you just listen for "eventStateChanged" in eventList and re-render when that happens that should fix it

**Review by skratz17 — APPROVED** [review]
Perf 👍

**Pull request merged**